tools/hermes/evaluate-line.js

```diff
@@ -14,15 +14,28 @@ C_STRING((function() {
   var HermesPromise = globalThis.Promise;
 
   function populateColors() {
-    colors.red = '\033[31m';
-    colors.green = '\033[32m';
-    colors.yellow = '\033[33m';
-    colors.blue = '\033[34m';
-    colors.magenta = '\033[35m';
-    colors.cyan = '\033[36m';
-    colors.white = '\033[37m';
 
-    colors.reset = '\033[0m';
+    if (typeof _replterminaltype !== "undefined" && _replterminaltype === "android") {
```

Comment:
This is supposed to be a property on the global object, right?

```suggestion
    if (globalThis._replterminaltype === "android") {
```

Reply:
It worked without ! And i believe the undefined variables will be looked up in global object when in non-strict mode ! Anyways, i fixed the code to make it explicit in the latest iteration ..